dyninstAPI/src/dynProcess.C

```diff
@@ -1066,12 +1066,12 @@ bool PCProcess::detachProcess(bool cont = true) {
             }
         }
 
-        tracedSyscalls_->removePreFork();
-        tracedSyscalls_->removePostFork();
-        tracedSyscalls_->removePreExec();
-        tracedSyscalls_->removePostExec();
-        tracedSyscalls_->removePreExit();
-        tracedSyscalls_->removePreLwpExit();
+        //tracedSyscalls_->removePreFork();
```

Comment:
@ManojKumarChauhan Can you explain why you need to remove these lines?

Reply:
In case, we attach to running process for more than one time, application was crashing due to deleting pre/post fork.